cluster/router/chain/chain.go

```diff
@@ -79,6 +103,99 @@ func (c *RouterChain) AddRouters(routers []router.PriorityRouter) {
 	c.routers = newRouters
 }
 
+// SetInvokers receives updated invokers from registry center. If the times of notification exceeds countThreshold and
+// time interval exceeds timeThreshold since last cache update, then notify to update the cache.
```

Comment:
// If the times of notification exceeds countThreshold and time interval exceeds timeThreshold since last cache update

why the condition type is 'and' which is not 'OR'?

Reply:
This is because I want to make sure cache rebuilding happens as less as possible, that is, I need to make sure cache to rebuild only once when at least `countThreshold` times of address notifications happen within the given time window (specified by `timeThreshold`